.github/actions/markdown-lint/action.yml

```
@@ -5,6 +5,11 @@ inputs:
     required: false
     description: "relative path to a .markdownlint.yaml config file"
     default: ".markdownlint.yaml"
+  folder_path:
+    required: false
+    description: "Specific folder to link"
```

[typo] `link` => `lint`
[optional] The suggestion below is based on https://github.com/gaurav-nelson/github-action-markdown-link-check#custom-variables 


```suggestion
    description: >
      By default the github-action-markdown-link-check action checks for all
      markdown files in this repo. Use this option to limit checks to only
      specific folders. Use comma separated values for checking multiple
      folders.
```